tests/performance/set_index.xml

```diff
@@ -3,17 +3,17 @@
     <fill_query>INSERT INTO test_in SELECT number FROM numbers(500000000)</fill_query>
 
     <!-- IN is used at index analysis -->
-    <query>SELECT count() FROM test_in WHERE a IN (SELECT rand(1) FROM numbers(200000)) SETTINGS max_rows_to_read = 1, read_overflow_mode = 'break'</query>
+    <query short="1">SELECT count() FROM test_in WHERE a IN (SELECT rand(1) FROM numbers(200000)) SETTINGS max_rows_to_read = 1, read_overflow_mode = 'break'</query>
 
-    <query>SELECT count() FROM test_in WHERE toInt64(a) IN (SELECT toInt64(rand(1)) FROM numbers(200000)) settings max_rows_to_read=1, read_overflow_mode='break'</query>
+    <query short="1">SELECT count() FROM test_in WHERE toInt64(a) IN (SELECT toInt64(rand(1)) FROM numbers(200000)) settings max_rows_to_read=1, read_overflow_mode='break'</query>
```

Comment:
This tests were checking the performance of SET creation. We need to fix it a little bit.

Reply:
I could switch it to max_rows_to_read=200000. That way it will still overflow but it will do it after executing the subquery.